### PR TITLE
Package tensorboard.0.0.1

### PR DESCRIPTION
**Pull request by LaurentMazare**
### `tensorboard.0.0.1`

---
* Homepage: https://github.com/LaurentMazare/ocaml-tensorboard
* Source repo: git+https://github.com/LaurentMazare/ocaml-tensorboard.git
* Bug tracker: https://github.com/LaurentMazare/ocaml-tensorboard/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

File: packages/tensorboard/tensorboard.0.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+maintainer:   "Laurent Mazare <lmazare@gmail.com>"
+authors:      [ "Laurent Mazare" ]
+
+build: [["dune" "build" "--only-packages" "tensorboard" "--root" "." "-j" jobs "@install"]]
+depends: [
+  "base" {>= "0.11.0"}
+  "bigstringaf"
+  "stdio"
+  "ocaml" {>= "4.06"}
+  "ocamlfind" {build}
```
Comment (kit-ty-kate) [inline packages/tensorboard/tensorboard.0.0.1/opam]: What do you need ocamlfind for?

Comment (kit-ty-kate) [reply packages/tensorboard/tensorboard.0.0.1/opam]: I can't find any mention of ocamlfind in your source code

File: packages/tensorboard/tensorboard.0.0.1/opam [hunk 2]
```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+bug-reports:  "https://github.com/LaurentMazare/ocaml-tensorboard/issues"
+homepage:     "https://github.com/LaurentMazare/ocaml-tensorboard"
+dev-repo:     "git+https://github.com/LaurentMazare/ocaml-tensorboard.git"
+maintainer:   "Laurent Mazare <lmazare@gmail.com>"
+authors:      [ "Laurent Mazare" ]
+
+build: [["dune" "build" "--only-packages" "tensorboard" "--root" "." "-j" jobs "@install"]]
```
Comment (kit-ty-kate) [inline packages/tensorboard/tensorboard.0.0.1/opam]: Could you use the more standard `["dune" "build" "-p" name "-j" jobs]` if it's ok for you?

File: packages/tensorboard/tensorboard.0.0.1/opam (hunk 1 above)
Comment (LaurentMazare) [reply packages/tensorboard/tensorboard.0.0.1/opam]: Good catch, I just removed this thanks.

File: packages/tensorboard/tensorboard.0.0.1/opam (hunk 2 above)
Comment (LaurentMazare) [reply packages/tensorboard/tensorboard.0.0.1/opam]: Fixed, indeed this looks simpler.

**Comment by mseri** [comment]
Thanks

File: packages/tensorboard/tensorboard.0.0.1/opam [hunk 3]
```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+bug-reports:  "https://github.com/LaurentMazare/ocaml-tensorboard/issues"
+homepage:     "https://github.com/LaurentMazare/ocaml-tensorboard"
+dev-repo:     "git+https://github.com/LaurentMazare/ocaml-tensorboard.git"
+maintainer:   "Laurent Mazare <lmazare@gmail.com>"
+authors:      [ "Laurent Mazare" ]
+
+build: [["dune" "build" "-p" name "-j" jobs]]
+depends: [
+  "base" {>= "0.11.0"}
```
Comment (kit-ty-kate) [inline packages/tensorboard/tensorboard.0.0.1/opam]: This was wrong. It should be `v0.11.0` not `0.11.0`. I'll fix that here directly. Could you fix it upstream too?

Comment (LaurentMazare) [reply packages/tensorboard/tensorboard.0.0.1/opam]: Thanks!

**Pull request merged**